csaf_2.1/json_schema/aggregator_json_schema.json

```diff
@@ -1,5 +1,5 @@
 {
-  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$schema": "https://docs.oasis-open.org/csaf/csaf/v2.1/meta_json_schema.json",
```

Comment:
Now that we are changing so many things, I prefer we drop the json in the stem. I think also, kebab-style would be best for the schema files. We should even be brave and drop the schema nonsense from the stem and maybe instead use a schema directory also on docs.oasis-open.org. That should be no problem:

```
https://docs.oasis-open.org/csaf/csaf/v2.1/schema/aggregatro.json
https://docs.oasis-open.org/csaf/csaf/v2.1/schema/csaf.json
https://docs.oasis-open.org/csaf/csaf/v2.1/schema/meta.json
https://docs.oasis-open.org/csaf/csaf/v2.1/schema/provider.json
```

Reply:
After I see, what that has done to our PR, I wished we would have done that in a separate PR. :see_no_evil: